src/Utils.jl

```diff
@@ -64,7 +64,7 @@ function plotBranchCont(contres::ContResult{T}, sol::M, contparms, plotuserfunct
 
 	# add the bifurcation points along the branch
 	if length(contres.bifpoint)>1
-		scatter!(map(x->x[2],contres.bifpoint[2:end]), map(x->x[4],contres.bifpoint[2:end]), label="", color = map(x->colorbif[x[1]],contres.bifpoint[2:end]), markersize=3, markerstrokewidth=0, subplot=3) |> display
+		scatter!(map(x->x[2],contres.bifpoint[2:end]), map(x->x[4],contres.bifpoint[2:end]), label="", color = map(x->colorbif[x[1]],contres.bifpoint[2:end]), markersize=3, markerstrokewidth=0, subplot=3)
```

Comment:
removed the `|> display` pipe so that it does not cause flashing with the plotting callback in `Flux.train()`
although travis does not seem to like this?

Reply:
It seems legit! Thank you